dp-cluster-setup-utility.py

```diff
@@ -423,6 +423,17 @@ def enable_trusted_proxy_for_atlas(self):
       'atlas.proxyuser.%s.groups' % knox_user: '*',
     }, note='updated by dp-cluster-setup-utility')
 
+  def enable_trusted_proxy_for_beacon(self):
+    if not self.cluster.has_service('BEACON'):
+      return
+    print 'Enabling Knox Trusted Proxy Support in BEACON...'
+    knox_user = self.cluster.knox_user()
+    print 'Setting trusted proxy configurations in beacon-security-site'
+    self.cluster.update_config('beacon-security-site', {
+      'trustedProxy.enabled' : 'true',
```

Comment:
We are not adding other properties like knox_user, trusted hosts and groups etc. Shouldn't we be adding them?

Reply:
Initially i was not aware that we can make these changes using this script so made the changes in beacon mpack. Will make the changes here and remove from beecon.